### PR TITLE
Story: [CCLS 1943] [CCLS 1937] [CCLS 2302] Upload & Edit Notification Attachments

### PR DESCRIPTION
**Pull request by farrell-m**

### CODE REVIEW

**Comment by farrell-m** [comment]
Base will be updated to `main` when https://github.com/ministryofjustice/laa-ccms-caab/pull/252 is merged.

File: src/main/resources/templates/notifications/upload-notification-attachment.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <div th:if="${attachmentUploadFormData.sendBy.code} == 'E'" class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-visually-hidden">Warning</span>
+          We can only accept <span th:text="${validExtensions}"/> files that are less than <span th:text="${maxFileSize}"/>.
+        </strong>
+      </div>
+      <p class="govuk-body-l">
+        Please provide details of the document you will be <span
+          th:text="${attachmentUploadFormData.sendBy.code} == 'P' ? 'posting' : 'uploading'"></span>.
```
Comment (PhilDigitalJustice) [inline src/main/resources/templates/notifications/upload-notification-attachment.html:29]: When accessing the form data you can accesss the data like this:

th:text="*{sendBy.code} == 'P' ? 'posting' : 'uploading'"></span>.

Just shortens the template. this works by using the form - th:object="${attachmentUploadFormData}"

Comment (farrell-m) [reply src/main/resources/templates/notifications/upload-notification-attachment.html]: Thanks, updated.

**Review by PhilDigitalJustice — APPROVED** [review]
LGTM

**Pull request merged**